### DISCUSSION TITLE
[2023-02-13] Component Browser Demo

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#5648] [Weekly Demo Day]
* **Hiding entries marked as PRIVATE**: Merged first part, and finished a flag toggling their visibility.
* **Searcher improvements**: 
   * The first step is done, where we display names first, and try to match entries also by the inserted code (so writing the function name with the underscore will work).
   * The old, postponed work on making CB understand the Enso language more was resumed.
* **Open Project Dialog** was restored. Still have some issues, but should be finished soon.
* **Fixes:**
  * The default Table visualization for Table-like types is merged.
  * The fix for duplicated entry on visualizations list is merged.
  * The fix for visualization being immediately hidden after pressing space is implemented and should be merged soon.
  * Merged fix for Cmd+click undoing the text
* **For the next week:**
  * Finishing rolling back to last save
  * Continue improving searching for entries: we will check why not the best match is put on the bottom and selected.
  * Finishing Table visualization: the API PR is about to be merged, then the actual 
  
  @vitvakatu @galin-enso please put screencasts of your demos.

**Comment by galin-enso** [comment]
Toggling PRIVATE entries visibility:

https://user-images.githubusercontent.com/117099775/218528515-b7394e7c-f06d-448e-8541-65c18e787791.mp4

Searcher entry matching improvements and method names before type names:

https://user-images.githubusercontent.com/117099775/218528553-c7f404be-661f-433b-9789-f00626282b19.mp4

**Comment by vitvakatu** [comment]
Opening a project using `cmd-O`:

https://user-images.githubusercontent.com/6566674/218718789-21bdb3c2-47c5-4302-87ff-ef95180ca4a1.mp4
